lib/models/receipts/receipt.dart

```diff
@@ -21,20 +21,22 @@ class Receipt {
   });
 
   /// Creates a receipt from a used ticket DTO
-  Receipt.fromTicketDTO(TicketDto dto)
+  Receipt.fromTicketResponse(TicketResponse dto)
       : productName = dto.productName,
         transactionType = TransactionType.ticketSwipe,
-        timeUsed = dto.dateUsed,
+        timeUsed = dto.dateUsed ?? DateTime.now(),
+        // fixme set better default
```

Comment:
should be resolved?

Reply:
I guess the question is whether we prefer this approach, or just forcing the value using ! Considering all receipts should have a timeUsed

Reply:
i meant to point out the comment

// fixme set better default

Reply:
That's an older issue related to #384. The solution would likely be to refactor how receipts works, by having a simple and a detailed one, though I think that is a matter for another PR. Or do you want it addressed now?

Reply:
im working on this in another PR so it should be fine